test/format/ops.c

```diff
@@ -652,10 +667,7 @@ ops(void *arg)
 		 */
 		if (!SINGLETHREADED &&
 		    !intxn && mmrand(&tinfo->rnd, 1, 100) >= g.c_txn_freq) {
-			iso_config = isolation_config(&tinfo->rnd, session);
-			testutil_check(
-			    session->begin_transaction(session, NULL));
-
+			iso_config = begin_transaction(tinfo, session);
```

Comment:
I'd prefer if `begin_transaction` was slightly renamed because I totally read it as `session->begin_transaction` in my head. Anything like `begin_transaction_iso` or `begin_transaction_config` or `start_transaction`.

Reply:
I think reading it as `WT_SESSION.begin_transaction` is OK, that's actually what it does (and the name matches the function `commit_transaction` that we use to commit).

I'll make setting `iso_config` a side-effect rather than a return value, that should help.